tsconfig.json

```diff
@@ -26,18 +25,22 @@
     },
 
     "types": [
-      "@types/node",
+      "node",
       "vitest/globals"
     ]
   },
 
   "include": [
     "docs",
-    "docs/.vitepress",
+    "docs/.vitepress/**/*",
     "lib",
+    "stories",
     "tests",
-    "env.d.ts",
-    "histoire.config.*",
-    "vitest.config.*"
+    "*.config.ts"
+  ],
+
+  "exclude": [
+    "node_modules",
+    "**/dist/**"
   ]
```

Comment:
Including `docs/.vitepress/**/*` ensures the compiler can pick up the files within the _hidden_ directory as `docs/.vitepress` alone does not get resolved. Up until now, `config.ts` was never visible to the compiler.

Excluding `dist` is no longer sufficient since we've offered the compiler a wildcard glob. `**/dist/**` ensures all nested dist directories (and there contents, in the case of `.vitepress`) are properly excluded.

Furthermore, it's safe to omit `env.d.ts` since it doesn't exist (or has been renamed for the time being). Can be added later at a more relevant time. `*.config.ts` removes the need to explicitly add individual files as they all share the same naming convention.

Reply:
Nice!

Reply:
As a side, this cumbersome _include_ list results from `.vitepress`. A much cleaner resolution would be:

```json5
// tsconfig.json
{
  "include": ["."],
  "exclude": ["node_modules", "dist"]
}
```
And be explicit where it's needed:
```json5
// docs/tsconfig.json
{
  "extends": "../tsconfig.json",
  "include": ".vitepress/**/*"
}
```

This way or the revised version in this PR work just the same. 👍

Reply:
Ah, clever 👀  But yeah I think it's OK to be explicit so. I would say the current version is fine 🚀